common/network-common/src/main/java/org/apache/spark/network/protocol/ChunkFetchFailure.java

```diff
@@ -54,7 +57,7 @@ public static ChunkFetchFailure decode(ByteBuf buf) {
 
   @Override
   public int hashCode() {
-    return Objects.hashCode(streamChunkId, errorString);
+    return Objects.hash(streamChunkId, errorString);
```

Comment:
Work-alikes from java.util; should do the same thing as it calls Arrays.hashCode in both cases